contracts/integrations/test/framework/assertions/stake.ts

```diff
@@ -50,33 +39,34 @@ export function validStakeAssertion(
         },
         after: async (
             expectedBalances: LocalBalanceStore,
-            _result: FunctionResult,
+            result: FunctionResult,
             args: [BigNumber],
             txData: Partial<TxData>,
         ) => {
+            // Ensure that the tx succeeded.
+            expect(result.success, `Error: ${result.data}`).to.be.true();
+
             const [amount] = args;
+            const { balanceStore, currentEpoch } = simulationEnvironment;
 
             // Checks that the ZRX transfer updated balances as expected.
             await balanceStore.updateErc20BalancesAsync();
             balanceStore.assertEquals(expectedBalances);
 
+            // _increaseCurrentAndNextBalance
+            loadCurrentBalance(ownerStake[StakeStatus.Undelegated], currentEpoch, true);
+            ownerStake[StakeStatus.Undelegated].currentEpochBalance = ownerStake[
+                StakeStatus.Undelegated
+            ].currentEpochBalance.plus(amount);
+            ownerStake[StakeStatus.Undelegated].nextEpochBalance = ownerStake[
+                StakeStatus.Undelegated
+            ].nextEpochBalance.plus(amount);
+
             // Checks that the owner's undelegated stake has increased by the stake amount
             const ownerUndelegatedStake = await stakingWrapper
-                .getOwnerStakeByStatus(txData.from!, StakeStatus.Undelegated) // tslint:disable-line:no-non-null-assertion
-                .callAsync();
-            const expectedOwnerUndelegatedStake = expectedUndelegatedStake(ownerStake, amount);
-            expect(ownerUndelegatedStake, 'Owner undelegated stake').to.deep.equal(expectedOwnerUndelegatedStake);
-            // Updates local state accordingly
-            ownerStake[StakeStatus.Undelegated] = expectedOwnerUndelegatedStake;
-
-            // Checks that the global undelegated stake has also increased by the stake amount
-            const globalUndelegatedStake = await stakingWrapper
```

Comment:
Why remove the global checks?

Reply:
we aren't actually updating the global undelegated stake in `stake` and `unstake` right? I need to investigate why this check was passing before actually :p

Reply:
o wait I'm trippin
back they go